.dev/systemimage/climate_machine_image.jl

```diff
@@ -11,28 +11,26 @@ sysimage_path =
 
 using Pkg
 Pkg.add("PackageCompiler")
+
 using PackageCompiler
 Pkg.activate(joinpath(@__DIR__, "..", ".."))
 Pkg.instantiate()
 pkgs = Pkg.installed()
 delete!(pkgs, "Pkg")
 
-# "TF_BUILD" is check to see if we are running on Azure for CI
-if !haskey(ENV, "TF_BUILD")
-    PackageCompiler.create_sysimage(
-        [Symbol(s) for s in keys(pkgs)];
-        sysimage_path = sysimage_path,
-        precompile_execution_file = joinpath(
-            @__DIR__,
-            "..",
-            "..",
-            "test",
-            "Numerics",
-            "DGmethods",
-            "Euler",
-            "isentropicvortex.jl",
-        ),
-    )
-else
-    return true
-end
+PackageCompiler.create_sysimage(
+    [Symbol(s) for s in keys(pkgs)];
```

Comment:
From @jkozdon  in https://github.com/CliMA/ClimateMachine.jl/pull/1056#issuecomment-624845226
> For users (as opposed to folks not hacking the CLIMA package itself), you could include a version that just depends on CLIMA itself, though there you would want to use as a precompile script something that uses the driver.

We might want to do this for the tests. From what I understand, this would just be:
```suggestion
    [:ClimateMachine];
```
but we probably want an option to do both.

Reply:
I agree. We want both behavior.

Reply:
add an option that can be checked by ARGS?

Reply:
added something simple for now (define path and add `true`)